Formula/couchdb.rb

```diff
@@ -60,15 +67,63 @@ def install
     system "make"
     system "make", "install"
 
+    install_geocouch if build.with? "geocouch"
+
     # Use our plist instead to avoid faffing with a new system user.
     (prefix/"Library/LaunchDaemons/org.apache.couchdb.plist").delete
     (lib/"couchdb/bin/couchjs").chmod 0755
-    (var/"lib/couchdb").mkpath
-    (var/"log/couchdb").mkpath
-    (var/"run/couchdb").mkpath
+  end
+
+  def geocouch_share
+    share/"couchdb-geocouch"
+  end
+
+  def install_geocouch
```

Comment:
Are you sure about that? Inlining this makes `install` long enough that it doesn't fit on a single screen, and with the extra indentation it's a little hard to see the control/conditionality flow. I'm guessing that most `couchdb` users will not be using `geocouch`, so it seems like it would be nicer to preserve the main install case as a single more readable method. Also, the scoping makes sure that the main install case doesn't depend on any local variables computed inside `install_geocouch`.